OpenRA.Mods.Common/Widgets/Logic/ModBrowserLogic.cs

```diff
@@ -115,9 +115,7 @@ public ModBrowserLogic(Widget widget, ModData modData)
 				catch (Exception) { }
 			}
 
-			Manifest initialMod;
-			Game.Mods.TryGetValue(Game.Settings.Game.PreviousMod, out initialMod);
-			SelectMod(initialMod != null && initialMod.Id != "modchooser" ? initialMod : Game.Mods["ra"]);
+			SelectMod(Game.Mods["ra"]);
```

Comment:
This was a temporary stopgap.  The entire file gets removed in the next commit :smile: